apps/ledger-live-desktop/tests/specs/onboarding/connect-device.spec.ts

```diff
@@ -25,6 +25,7 @@ test.describe.parallel("Onboarding", () => {
       await test.step("Get started", async () => {
         await onboardingPage.getStarted();
         await onboardingPage.hoverDevice(Nano.nanoS);
+        await new Promise(r => setTimeout(r, 300)); // wait for the animation to finish
```

Comment:
before this change, I ran /generate-screenshots and it generated a screenshots where the hover wasn't visible